### PR TITLE
font-display: swap for google fonts (for dartdoc pages)

### PR DESCRIPTION
**Pull request by isoos**
#5261

### CODE REVIEW

**Review by jonasfj — APPROVED** [review]

File: app/lib/dartdoc/customization.dart [hunk 1]
```diff
@@ -75,6 +75,7 @@ class DartdocCustomizer {
     _addAlternateUrl(head, canonical);
     _addAnalyticsTracker(head, body);
     _addGithubMarkdownStyle(head, body);
+    _replaceFontUrls(head);
```
Comment (jonasfj) [inline app/lib/dartdoc/customization.dart:78]: Why is this needed?
https://github.com/dart-lang/dartdoc/blob/master/lib/templates/html/_head.html#L25

Is it just we want it for material icons too?

**Comment by isoos** [comment]
Closing, as it is not recommended for icons to use the font-display: swap setting.
https://www.zachleat.com/web/font-display-icon-fonts/
